#goal
uint8 RAISE_ARM = 0
uint8 WAVE_ARM = 1
uint8 POINT_FORWARD = 2

uint8 armMovementType
duration gesture_duration

---
#state

---
#feedback

uint8 MOVING_TO_TARGET = 0
uint8 AT_TARGET = 1
uint8 feedbackState
